docs/handbook/01_intro.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Quantum computing has the potential to revolutionize many fields in the 21st century. Over the past decade, numerous quantum computers from multiple providers based on different qubit technologies have been made publicly available.
+However, the best hardware is only as good as the software available to realize corresponding applications on it---a lesson learned from the past decades of research on designing and developing classical circuits and systems.
+Thanks to the software tools and methods for _Electronic Design Automation (EDA)_, we can create classical systems with a staggering amount of transistors and complex functionalities that we often take for granted.
+These methods allow designers to efficiently and automatically handle the intricacies of such systems and optimize their performance.
+Compared to that, most existing software solutions for quantum computing are based on manual approaches.
```

Comment:
As before, shouldn't we cite something to support these claims?

Reply:
I rewrote the sentence here a little bit and added a citation. However, I find it rather hard to find proper references here.
So could you please check that part again and make a suggestion if you are not yet satisfied with the current way this is written?

Reply:
I would still like to keep this kind of sentence. The question to me then is whether we just leave out the reference (which would be fine by me), we accept the current reference, or we find some additional ones.
To be fair, I am slightly leaning towards the first option.

Reply:
I get the sentiment. We want to say how MQT stands out from other quantum software packages. The USP is the EDA expertise which we mentioned before. But this sentence makes it seem as if other software packages don't do any automation. I am in favor of leaving it out. But it is ultimately not that important either.

Reply:
One more adjustment to the sentence. Current version now is:

> Compared to that, most existing software solutions for quantum computing leave the decades of research on design automation methods underutilized---making the resulting tools susceptible to errors, inefficiency, and inconsistency.

I think this is a fair statement now. Would you also be fine with that?

Reply:
I think that's good. Sorry for nagging around and thanks for dealing with our inputs :smile:

Reply:
No worries your input is much appreciated 🙏